ep/build-velox/src/get_velox.sh

```diff
@@ -16,8 +16,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=2025_04_16
+VELOX_REPO=https://github.com/wecharyu/velox.git
+VELOX_BRANCH=support_to_json
```

Comment:
To test with your PR,  you could pick the patch to your own branch based on the latest Velox branch used in Gluten. Please make sure the only difference is your patch is additionally applied.

https://github.com/apache/incubator-gluten/blob/main/ep/build-velox/src/get_velox.sh#L20